datastore/multi.go

```diff
@@ -12,21 +12,23 @@ package datastore
 
 import (
 	ds "github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-datastore/namespace"
 )
 
 var (
 	// Individual Store Keys
-	rootStoreKey   = ds.NewKey("/db")
-	systemStoreKey = rootStoreKey.ChildString("/system")
-	dataStoreKey   = rootStoreKey.ChildString("/data")
-	headStoreKey   = rootStoreKey.ChildString("/heads")
-	blockStoreKey  = rootStoreKey.ChildString("/blocks")
+	rootStoreKey   = ds.NewKey("db")
```

Comment:
question: Is this a bug fix?  Were we adding `//` to our keys instead of `/` here?

Reply:
`ChildString` does `NewKey(k.string + "/" + s)` and the `NewKey` function adds a `/` prefix if none are present and cleans the `//` that would have been present. Removing the `/` from the passed param makes it more "correct" according to what the function is doing.

Reply:
Ah nice - thanks Fred :)